doc/update_source_files_to_2.0.0.sed

```diff
@@ -0,0 +1,37 @@
+# A script for the stream editor sed to update .cc and .h files from the
+# naming scheme used in ASPECT 1.5.0 to ASPECT 2.0.0. This script correctly
+# updated all files within the official development version of ASPECT,
+# but it is not guaranteed to work for all possible names in user plugins.
+# Consequently a backup of files is strongly recommended, and the changes
+# created by this script should be investigated to ensure a correct renaming.
+#
+# Usage for a file named FILENAME: 
+# sed -i -f update_source_files.sed FILENAME
```

Comment:
can you say what `FILENAME` is supposed to be here? A single source file? Can it contain wildcards?